src/tests/api/e2eHappyPath.test.ts

```diff
@@ -19,10 +19,7 @@ describe("E2E Happy Path Tests", () => {
 		[dataSlim],
 		[dataBjorn],
 	])("photo ID journey", async (userData: any) => {
-		const sessionResponse = await startStubServiceAndReturnSessionId(userData.journeyType);
-		expect(sessionResponse.status).toBe(200);
```

Comment:
Nice yeah I agree this wasn't really needed since it's not actually the service under test 👍